### ISSUE TITLE
(Wii) Allow changing video width for custom resolutions

### ISSUE THREAD

**Issue by SuperrSonic** [libretro/RetroArch#598]
I would like to test some custom resolutions with GBA games and 384, 512, 530, 640 is not enough, instead of asking for yet another custom resolution(608x480) perhaps we could get the option of letting the user change width scaling for custom viewports??

I made a thread in the forums that explains why I need to use 608 and why 640 does not give the desired output. http://www.libretro.com/forums/viewtopic.php?f=31&t=1545&p=12167
In short 530 gave me a good result but it was too stretched, 608 would provide a more correct scale for GBA games, in accordance to the GB Player.
